cmake/MiNiFiOptions.cmake

```diff
@@ -82,6 +80,8 @@ list(APPEND STRICT_GSL_CHECKS_Values AUDIT ON DEBUG_ONLY OFF)
 set_property(CACHE STRICT_GSL_CHECKS PROPERTY STRINGS ${STRICT_GSL_CHECKS_Values})
 
 if (WIN32)
+    add_minifi_option(INSTALLER_MERGE_MODULES "Creates installer with merge modules" OFF)
+    add_minifi_option(INSTALLER_WITH_VC_REDISTRIBUTABLES "Creates installer with Visual C++ redistributables included" OFF)
```

Comment:
The naming is confusing, and not specific enough. Merge modules and bundled DLLs are both Visual C++ redistributable runtimes (runtime meaning library or DLL in this case), they're just different ways of installing the same DLLs of the C and C++ standard libraries. Under the old version, if the user didn't ask for merge modules, they got bundled DLLs.

There is also the Universal C Runtime (UCRT) DLLs that are not part of the same Visual C++ redistributable runtimes package, but they are also standard library DLLs, they also fall under the same license terms, and are not redistributable under the Apache License. If someone wants to NOT include the Visual C++ redistributable runtime, then they probably want a package free of Microsoft blobs, and they don't want to include the UCRT either.

The previous version was fine, as it treated both kinds of redistributables under the same banner, disabling them altogether. This new version seems to treat the Visual C++ redistributable runtime package separately from the UCRT, and I don't see the rationale behind this.

Reply:
I agree the naming should be changed, to be more clear, and clarify which option bundles what artifacts exactly. It also needs to be specified if a bundle falls under a different license.

On the other hand I would keep all three bundle options separate. The rationale behind this version is it's easier and more direct to specify a single option for each installer bundle option instead of combining build options like `INCLUDE_MS_BLOBS` + `USE_UCRT_DLLS`. So I would keep them separately like `INCLUDE_UCRT_IN_INSTALLER`, `INCLUDE_VC_REDIST_DLLS`, and `INCLUDE_VC_REDIST_MERGE_MODULES`. This way you can directly specify which libraries you want to include in the installer, and by not specifying anything, no Microsoft blobs will be included. What are your thoughts on this?

Reply:
This sounds good to me

Reply:
Updated the flag names and descriptions in b2da5ed65397525433093fa654edfe0d7836ca13 also making them mutually exclusive